cluster/src/main/java/org/apache/iotdb/cluster/query/ClusterPlanExecutor.java

```diff
@@ -307,7 +307,12 @@ protected int getNodesNumInGivenLevel(PartialPath path, int level) throws Metada
     // if path doesn't end with **.
     // e.g. we have SG root.sg.a and root.sg.b, the query path is root.sg, we should return the map
     // with key root.sg.a and root.sg.b instead of an empty one.
-    PartialPath wildcardPath = path.concatNode(IoTDBConstant.MULTI_LEVEL_PATH_WILDCARD);
+    PartialPath wildcardPath;
+    if (!path.getMeasurement().equals(IoTDBConstant.MULTI_LEVEL_PATH_WILDCARD)) {
+      wildcardPath = path.concatNode(IoTDBConstant.MULTI_LEVEL_PATH_WILDCARD);
+    } else {
+      wildcardPath = path;
+    }
```

Comment:
```suggestion
    PartialPath wildcardPath = path;
    if (!wildcardPath.getMeasurement().equals(IoTDBConstant.MULTI_LEVEL_PATH_WILDCARD)) {
      wildcardPath = wildcardPath.concatNode(IoTDBConstant.MULTI_LEVEL_PATH_WILDCARD);
    }
```

Reply:
Thanks for your comments. The codes have been refined. Does this PR need to be separeted to different ones? If so, I will close this one.

Reply:
Pay attention to it next time is ok~